lib/perl/Genome/Model/Tools/Bedpe.pm

```diff
@@ -0,0 +1,30 @@
+package Genome::Model::Tools::Bedpe;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::Bedpe {
+    is => ['Command'],
```

Comment:
Maybe?  Seems to work as is though:
> gmt bedpe -h
Using libraries at /gscuser/aregier/git/genome/copy/lib/perl
Sub-commands for gmt bedpe:
 evaluate-bedpe  --bedpe=? --gold-...  no description!!!: define 'doc' in the class definition for
                                        Genome::Model::Tools::Bedpe::EvaluateBedpe